dephell_setuptools/_pkginfo.py

```diff
@@ -17,6 +17,15 @@ def content(self) -> Dict[str, Any]:
         cmd = ['pkginfo', '--json', str(self.path)]
         result = subprocess.run(cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
         if result.returncode != 0:
-            raise RuntimeError(result.stderr.decode().split('\n')[-1])
-        content = json.loads(result.stdout.decode())
+            msg_lines = result.stderr.decode().rstrip().split('\n')
+            raise RuntimeError(msg_lines[-1] if msg_lines else 'Unknown error')
+        stdout = result.stdout.decode()
+        if not stdout:
+            return {}
+
+        try:
+            content = json.loads(stdout)
+        except json.decoder.JSONDecodeError:
+            return {}
```

Comment:
Why this should be silenced? All exceptions from readers go into `read_setup` where they are handled by callback function.

Reply:
This one was causing exceptions reading invalid json from pkginfo to be shown in the console, usually when pkginfo was emitting nothing, which isnt valid json.

Reply:
Good point :)